pages/contributions/_id.vue

```diff
@@ -11,11 +11,11 @@
       </template>
       <div class="details">
         <Tag mr2>{{ contribution.category }}</Tag>
-        <p v-if="contribution.profile">
+        <p v-if="contribution.profile" class="icon-list">
```

Comment:
this is a paragraph, not a list so that's weird to have an icon-list class, there is probably something wrong in the design of that

Reply:
It's correct, like i said maybe the name icon-list is not the best. I want to have one global css class for this kind of style.